www_src/components/snackbar/snackbar.less

```diff
@@ -0,0 +1,17 @@
+.snackbar {
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: #444;
+  line-height: 3em;
+  text-align: center;
+  color: #EEE;
+  height: 3em;
```

Comment:
Try not to set explicit heights on elements that contain live text.

This leads to lots of problems when strings get longer (localization...etc).